cafebabel/templates/base.html

```diff
@@ -3,6 +3,21 @@
 <meta name=viewport content="width=device-width, minimum-scale=1, initial-scale=1, shrink-to-fit=no">
 <title>{% block meta_title %}pan-European news, features and opinions{% endblock %} – Cafebabel</title>
 <meta name=description content="{% block meta_description %}{% endblock %}">
+<!--Social-->
+<meta property="og:url" content="{{ request.url }}">
+<meta property="og:title" content="{{ self.meta_title() }}">
+<meta property="og:image" content="{% block meta_image_absolute_url %}{% endblock %}">
```

Comment:
There should be a default image, useful when a link to the homepage is shared. Same for other properties, which makes me think if it's pertinent to add meta for pages like edition and so on. Maybe being more specific than the base template is better?

Reply:
you are right. For exemple pages for edition have no need to be shared with meta

Reply:
In that case it should not be part of the base template but added as [a macro](http://jinja.pocoo.org/docs/2.10/templates/#macros)?

Reply:
ok, I'll do that